entries/1/123-reg.co.uk.json

```
@@ -1,10 +1,10 @@
 {
   "123 Reg": {
     "domain": "123-reg.co.uk",
-    "contact": {
-      "facebook": "123regfans",
-      "twitter": "123reg"
-    },
+    "tfa": [
+      "totp"
```

```suggestion
      "sms",
      "totp",
      "u2f"
```

According to the documentation page, they support SMS and FIDO2 security keys as well.